pkg/crc/config/settings.go

```diff
@@ -77,8 +89,15 @@ func RegisterSettings(cfg *Config) {
 		"Disable update check (true/false, default: false)")
 	cfg.AddSetting(ExperimentalFeatures, false, ValidateBool, SuccessfullyApplied,
 		"Enable experimental features (true/false, default: false)")
-	// shared dir is not implemented for windows yet
-	if runtime.GOOS == "darwin" || runtime.GOOS == "linux" {
+
+	// Shared directories configs
+	if runtime.GOOS == "windows" {
+		cfg.AddSetting(SharedDirPassword, "", ValidateString, SuccessfullyApplied,
+			"Password used while using CIFS/SMB file sharing (Its the password for the current logon user)")
+
+		cfg.AddSetting(EnableSharedDirs, false, validateSmbSharedDirs, SuccessfullyApplied,
+			"Enable shared directories which mounts host's $HOME at /home in the CRC VM (true/false, default: false)")
```

Comment:
why is `$HOME` used here, in Windows terminology this is:

 * [%USERPROFILE%](https://en.wikipedia.org/wiki/%25USERPROFILE%25)
 * [%HOMEDRIVE%](https://en.wikipedia.org/wiki/%25HOMEDRIVE%25)[%HOMEPATH%](https://en.wikipedia.org/wiki/%25HOMEPATH%25)

It would be best to use something familiar and accurate to the user; `$HOME` only exists as a derived variable in PowerShell. WDYT?

Reply:
yeah, help message is not accurate, would be better to just avoid using `env` variables and have the message in plain english..

how about ?
>Mounts host user's home directory or user profile folder in the CRC VM (true/false, default: false)